jsx/Form.js

```diff
@@ -1739,6 +1739,147 @@ class LorisElement extends Component {
   }
 }
 
+/**
+ * Radio Component
+ * React wrapper for a <input type='radio'> element.
+ */
```

Comment:
```suggestion
 *
 * Example `options` prop: 
 *   {
 *     female: 'Female',
 *     male: 'Male',
 *   }
 */
```

would be nice if we could show how to provide the options prop in the doc comments